### PR TITLE
Fix #22 - Flow Control

### LINKED ISSUE #22: sidik https://tour.golang.org/flowcontrol/{1-14}

### PR DESCRIPTION
**Pull request by Muh-Sidik**

### CODE REVIEW

File: sidik/basic_11.go [hunk 1]
```diff
@@ -8,7 +8,7 @@ import (
 var ( //membuat variabel bisa sekaligus dengan pembungkus
 	Tobe bool	= false
 	Maxint uint64 = 1<<64 - 1
-	zo	complex128 = cmplx.Sqrt(-5 + 12i)
+	zo	complex128 = cmplx.Sqrt(-5 + 12i) //type data complex128
```
Comment (Muh-Sidik) [inline sidik/basic_11.go:11]: belum tau type data complex128

Comment (muizidn) [reply sidik/basic_11.go:11]: **Complex Numbers**: The complex numbers are divided into two parts are shown in the below table. float32 and float64 are also part of these complex numbers. The in-built function creates a complex number from its imaginary and real part and in-built imaginary and real function extract those parts.

https://www.geeksforgeeks.org/data-types-in-go/

Comment (Muh-Sidik) [reply sidik/basic_11.go:11]: oke

**Review by muizidn — APPROVED** [review]
👍

File: sidik/flowcontrol_2.go [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package sidik
+
+import "fmt"
+
+func flow2()  {
+	sum := 1
+	for ; sum < 1000; {
```
Comment (muizidn) [inline sidik/flowcontrol_2.go:7]: The init and post statements are optional.

**Review by muizidn — DISMISSED** [review]

File: sidik/flowcontrol_3.go [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package sidik
+
+import "fmt"
+
+func flow3()  {
+	sum := 1
+	for sum < 1000 {
```
Comment (muizidn) [inline sidik/flowcontrol_3.go:7]: For is Go's "while"

File: sidik/flowcontrol_10.go [hunk 4]
```diff
@@ -0,0 +1,21 @@
+package sidik
+
+import (
+	"fmt"
+	"time"
+)
+
+func flow10()  {
+	fmt.Println("When is Saturday")
```
Comment (muizidn) [inline sidik/flowcontrol_10.go:9]: Switch evaluation order

File: sidik/flowcontrol_8.go [hunk 5]
```diff
@@ -0,0 +1,11 @@
+package sidik
+
+import "fmt"
+
+func Sqrt(x float64) float64  {
```
Comment (muizidn) [inline sidik/flowcontrol_8.go:5]: https://tour.golang.org/flowcontrol/8

File: sidik/flowcontrol_9.go [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+)
+
+func flow9()  {
+	fmt.Println("Go runs on")
+	switch os := runtime.GOOS; os {
+	case "darwin":
+		fmt.Println("OS X")
+	case "linux":
+		fmt.Println("Linux")
+	default:
```
Comment (muizidn) [inline sidik/flowcontrol_9.go:15]: "windows"?

File: sidik/sidik.go [hunk 7]
```diff
@@ -22,6 +22,21 @@ func Main() {
 	basic15()
 	basic16()
 	basic17()
+	fmt.Printf("\n")
```
Comment (muizidn) [inline sidik/sidik.go:25]: Lihat di PR @AlMishbah

Comment (muizidn) [reply sidik/sidik.go:25]: https://github.com/pondok-programmer/golang-learn/pull/32#discussion_r374586807

**Review by muizidn — APPROVED** [review]

**Comment by muizidn** [comment]
Remove WIP when this works is merge ready.

**Review by muizidn — APPROVED** [review]

**Pull request merged**